src/amo/components/HomeHero/styles.scss

```diff
@@ -0,0 +1,98 @@
+@import '~amo/css/styles';
+
+.HomeHero-header {
+  color: $black;
+  padding: 0 12px 12px;
+  text-align: center;
+
+  @include respond-to(medium) {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  @include respond-to(large) {
+    padding-left: 30px;
+    padding-right: 30px;
+  }
+}
+
+.HomeHero-title {
+  font-size: $font-size-xl;
+  line-height: 1;
+  margin: 24px 0 20px;
+}
+
+.HomeHero-subtitle {
+  font-size: $font-size-l;
+  font-weight: normal;
+  line-height: 1;
+  margin: 12px 0 24px;
+}
+
+.HomeHero-cards {
+  .Card-contents {
+    background: none;
+    padding: 0;
+  }
+
+  .Hero-contents {
+    grid-template: none;
+    grid-template-areas: none;
+    grid-template-columns: none;
+    margin-bottom: 8px;
+    padding: 0 12px;
+
+    @include respond-to(medium) {
+      padding-left: 24px;
+      padding-right: 24px;
+    }
+
+    @include respond-to(large) {
+      grid-gap: 30px;
+      grid-template-columns: 1fr 1fr 1fr;
+      margin-bottom: 30px;
+      padding-left: 30px;
+      padding-right: 30px;
+    }
+  }
+
+  h4 {
+    font-size: $font-size-m;
+    font-weight: normal;
+    margin: 12px 0;
+    text-align: center;
+  }
+}
+
+.HomeHero-card {
+  .Card-contents {
+    align-items: center;
+    background-color: $white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 24px 0 12px;
+
+    @include respond-to(large) {
+      min-height: 138px;
```

Comment:
Why 138 exactly?

Reply:
for ~tablet, the "Enhance your media Experience" makes the card expand because the text wraps to a 2nd line. I noticed it was approx this height. So I went with that so the other cards would match and it would be balanced.

Reply:
That's true for `en-US` but likely not for `de` (German) for instance.

Reply:
oh? will it be longer for de? I guess I didn't think of it being longer...

let me look into another approach then..